### ISSUE TITLE
neofs-cli object hash has no session flag

### ISSUE THREAD

**Issue by abereziny** [nspcc-dev/neofs-node#2029]
In order to use static session with neofs-cli we need a session flag to pass a file with session token.

```
neofs-cli object hash --help
```

## Expected Behavior
session flag should be in the list of available flags

## Current Behavior
session flag absent, cannot use `neofs-cli object hash` with static session

## Your Environment
```
neofs-cli --version
NeoFS CLI
Version: v0.33.0-33-g60e9de8d 
GoVersion: go1.18.4
```

## Autotest
in feature branch
https://github.com/abereziny/neofs-testcases/tree/feature/abereziny-add-object-static-session-tests
testsuites.session_token.test_object_session_token#test_static_session_range

**Comment by fyrchik** [comment]
Let's make all commands symmetric.

**Comment by fyrchik** [comment]
Closed via #2039 .
